src/root/views/Country/ThreeW/InCountryProjects/index.tsx

```diff
@@ -156,31 +187,31 @@ function InCountryProjects(props: Props) {
           </div>
           <Container
             className={styles.ongoingProject}
-            heading="Ongoing projects"
+            heading={viewAllProjects ? 'All Projects' : 'Ongoing Projects'}
             actions={(
               <Button
                 actions={<IoChevronForward />}
                 variant="tertiary"
-                disabled
+                onClick={toggleViewAllProject}
               >
-                View all projects
+                { viewAllProjects ? 'View Ongoing Projects' : 'View All Projects' }
```

Comment:
Add fixme to use string or use strings

Reply:
We'll be adding strings for FieldReport and 3W in bulk